lesson-one/examples/hello-world-dom.html

```diff
@@ -0,0 +1,30 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="dom.css"/>
+		<title>Dom's Space Merchandise</title>
+	</head>
+	<body>
+		<h1>Dom's Space Merchandise</h1>
+		<table border=1>
```

Comment:
While this is technically correct, it's better to have your styles seperate from your content